logstreams/src/test/java/io/camunda/zeebe/logstreams/impl/log/LogEntryDescriptorTest.java

```diff
@@ -31,9 +31,9 @@ public void shouldMarkAsProcessed() {
     final var buffer = new UnsafeBuffer(new byte[128]);
 
     // when
-    LogEntryDescriptor.markAsProcessed(buffer, 0);
```

Comment:
:wrench: : I prefer more methods which out boolean flags, like we had before. It is often unclear what the value of the boolean does, and sometimes you have only one usecase like we do here we always set it to true.